src/test/java/teammates/test/cases/browsertests/BaseUiTestCase.java

```diff
@@ -62,6 +49,18 @@ protected void releaseBrowser() {
     }
 
     /**
+     * Reminder to disable GodMode and re-run the test(s).
+     */
+    @AfterSuite
+    public static void remindUserToDisableGodModeIfRequired() {
+        if (TestProperties.IS_GODMODE_ENABLED) {
```

Comment:
Cool! GodMode-in-CI [fails within 7 minutes](https://travis-ci.org/TEAMMATES/teammates/builds/240293743)